lib/rubycritic/generators/json/simple.rb

```diff
@@ -24,6 +25,18 @@ def data
             score: @analysed_modules.score
           }
         end
+
+        def file_directory
+          @file_directory ||= Pathname.new(Config.root)
```

Comment:
At the moment user can configure saving directory by using -p "path/to/directory/", this feature is still working.